ssl/test/runner/runner.go

```diff
@@ -19704,27 +19749,94 @@ func addMultipleCertSlotTests() {
 		// ED25519 signature algorithm should be prioritized if supported.
 		ed25519PriorityTest := testCase{
 			testType: serverTest,
-			name:     prefix + "Multiple-Cert-ED25519-Priority" + suffix,
+			name:     "Server-Multiple-Cert-ED25519-Priority" + suffix,
 			config: Config{
 				MaxVersion:                ver.version,
 				VerifySignatureAlgorithms: allAlgorithms,
 			},
 			flags: append(
-				[]string{
-					"-multiple-certs-slot", path.Join(*resourceDir, getShimCertificate(testCertRSA)) + "," + path.Join(*resourceDir, getShimKey(testCertRSA)),
-					"-multiple-certs-slot", path.Join(*resourceDir, getShimCertificate(testCertECDSAP256)) + "," + path.Join(*resourceDir, getShimKey(testCertECDSAP256)),
-					"-multiple-certs-slot", path.Join(*resourceDir, getShimCertificate(testCertEd25519)) + "," + path.Join(*resourceDir, getShimKey(testCertEd25519)),
-				},
+				certificateSlotFlags,
 				flagInts("-curves", []int{int(CurveX25519), int(CurveP256)})...,
 			),
 			expectations: connectionExpectations{
 				peerSignatureAlgorithm: signatureEd25519,
 			},
 		}
 
+		// Below tests verify that multiple certificates only works because we have a valid cert.
+
+		// Connection shouldn't be accepted without RSA cert.
+		noRSACertTest := testCase{
+			testType: serverTest,
+			name:     "Server-Multiple-Cert-No-RSA" + suffix,
```

Comment:
Why doesn't this show up in the transfer test case name?

Reply:
I'm speculating that the transfer test case names only include test cases that succeed. These tests are testing for specific failures. We also only include tests on the server side.
https://github.com/aws/aws-lc/blob/c516faf2bf3f788ff842317e779fe13cf901acfd/ssl/test/runner/runner.go#L2052-L2059